src/frontend/components/Card.jsx

```diff
@@ -45,7 +48,7 @@ export function Card({
                 </div>
               </div>
             </div>
-            <button className="btn btn-sm">
+            <button className="btn btn-sm" onClick={() => addToCart(item)}>
```

Comment:
If the item is added to the cart, the button should show 'Go to Cart' and redirect to Cart on clicking it.